README.md

```diff
@@ -61,7 +71,27 @@ test('Add new item', async t => {
 });
 ```
 
-#### Limitations
+As an alternative, the .getVue() method can take a function that returns the required data. This function acts as a filter. Its argument is an object returned by .getVue(), i.e. { props: ..., state: ..., computed:...}.
```

Comment:
Please highlight all code in readme 
.getVue(), i.e. { props: ..., state: ..., computed:...} -> `.getVue()`, i.e. `{ props: ..., state: ..., computed:...}`
etc.